index.html

```diff
@@ -1,10 +1,73 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ru">
 <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
+    <meta name="description" content="homework" />
+    <meta name="keywords" content="lists tables and forms" />
+    <meta name="author" content="Alexey Tretyakov" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles/style.css">
     <title>lesson</title>
 </head>
 <body>
-<h1>lesson 1</h1>
+<div class="advantages">
+    <h2 class="heading">What do we offer</h2>
+    <hr class="highlight">
+<!--Var 2-->
+    <!--    <div class="content">-->
+    <!--      <div class="vectors">-->
+    <!--        <img class="vectors-img" src="./img/headphones.png" alt="Support 24/7">-->
+    <!--        <img  class="vectors-img" src="./img/communicate.png" alt="Communicate directly">-->
+    <!--        <img class="vectors-img" src="./img/bookonline.png" alt="Book online">-->
+    <!--        <img class="vectors-img" src="./img/like.png" alt="Real quest reviews">-->
+    <!--      </div>-->
+    <!--      <div class="img-text">-->
+    <!--        <p class="img-text-w">Support 24/7</p>-->
+    <!--        <p class="img-text-w">Communicate directly</p>-->
+    <!--        <p class="img-text-w">Book online</p>-->
+    <!--        <p class="img-text-w">Real quest reviews</p>-->
+    <!--      </div>-->
+    <!--    </div>-->
+    <!---->
+    <div class="content">
+
+        <div class="vectors">
+            <div class="down">
+                <img class="vectors-img" src="./img/headphones.png" alt="Support 24/7">
+            </div>
+            <div class="img-text">
```

Comment:
Why do you need a wrapper for this text? Please, don't add wrappers for wrappers, if you can. This is an excess code